lib/Taxonomy/ArtistCategory.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright (c) 2021. Geniem Oy
+ */
+
+namespace TMS\Theme\Filharmonia\Taxonomy;
+
+use \TMS\Theme\Base\Interfaces\Taxonomy;
+use TMS\Theme\Base\Traits\Categories;
+use TMS\Theme\Filharmonia\PostType\Artist;
+
+/**
+ * This class defines the taxonomy.
+ *
+ * @package TMS\Theme\Base\Taxonomy
+ */
+class ArtistCategory implements Taxonomy {
+
+    use Categories;
+
+    /**
+     * This defines the slug of this taxonomy.
+     */
+    const SLUG = 'artist-category';
+
+    /**
+     * Add hooks and filters from this controller
+     *
+     * @return void
+     */
+    public function hooks() : void {
+        add_action( 'init', \Closure::fromCallable( [ $this, 'register' ] ), 15 );
+    }
+
+    /**
+     * This registers the post type.
+     *
+     * @return void
+     */
+    private function register() {
+        $labels = [
+            'name'                       => 'Kategoriat',
+            'singular_name'              => 'Kategoria',
+            'menu_name'                  => 'Kategoriat',
+            'all_items'                  => 'Kaikki kategoriat',
+            'new_item_name'              => 'Lisää uusi kategoria',
+            'add_new_item'               => 'Lisää uusi kategoria',
+            'edit_item'                  => 'Muokkaa kategoriaa',
+            'update_item'                => 'Päivitä kategoria',
+            'view_item'                  => 'Näytä kategoria',
+            'separate_items_with_commas' => \__( 'Erottele kategoriat pilkulla', 'tms-theme-base' ),
```

Comment:
Onko nämä tarkoituksella laitettu käännettäviksi, vaikka muut stringit tässä ei? Jos näitä ei tarpeen kääntää, niin ehkä siis nuo pois, jotta ei turhaan tule po/mo-filuihin mukaan.